microsoft-advanced-threat-analytics/info.json

```diff
@@ -0,0 +1,144 @@
+        "description": "Specify the Microsoft Advanced Threat Analytics server URL to connect and perform the automated operations.",
+        "placeholder": "https://atacenter.contoso.com"
+      },
+      {
+        "name": "username",
+        "title": "Username",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "description": "Specify the Microsoft Advanced Threat Analytics username to connect and perform the automated operations."
+      },
+      {
+        "name": "password",
+        "title": "Password",
+        "type": "password",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "description": "Specify the Microsoft Advanced Threat Analytics password to connect and perform the automated operations.."
+      },
+      {
+        "name": "verify_ssl",
+        "title": "Verify SSL",
+        "type": "checkbox",
+        "editable": true,
+        "visible": true,
+        "required": false,
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified. \nBy default, this option is set to True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_suspicious_activities_list",
+      "title": "Get Suspicious Activities List",
+      "annotation": "get_suspicious_activities_list",
+      "description": "Retrieves the suspicious activities by the activity ID, status, severity from Microsoft Advanced Threat Analytics.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "activity_id",
+          "title": "Suspicious Activity ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the ID of the suspicious activity.",
+          "description": "Specify the ID of the suspicious activity to retrieve. If provided, all other arguments will be ignored."
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "set_suspicious_activity_status",
+      "title": "Set Suspicious Activity Status",
+      "annotation": "set_suspicious_activity_status",
+      "description": "Sets status of suspicious activity by the activity ID, status in Microsoft Advanced Threat Analytics.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "activity_id",
+          "title": "Suspicious Activity ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the ID of the suspicious activity.",
+          "description": "Specify the ID of the suspicious activity whose status needs to be updated."
+        },
+        {
+          "name": "status",
+          "title": "Status",
+          "type": "text",
```

Comment:
This parameter can be changed to select.

Reply:
We dont have setup also not API guide so kept it as text.